### DISCUSSION TITLE
Unable to view the security onion console on the browser

### DISCUSSION THREAD

**Discussion by ncampbell89** [Security-Onion-Solutions/securityonion#12498] [Unsupported Versions]
Hello everyone, I'm having issues trying to display the security onion log in page when I was putting in the IP address in the address bar. I've been using https and http, nothing has been working. I also tried clearing caches, going incognito, tried Google Chrome like the documentation said, tried bridged connections from my VM and nothing is working. Everything is already configured in the back end of security onion from my virtual machine. I checked so-status and everything says "OK". The web browser keeps timing out every time I put in the IP address provided to me by security onion.

**Comment by DebianGuru** [comment]
Way more information needed.  Version of SecurityOnion?  What kind of hypervisor on what OS?  How are your NICs configured (Maybe your NICs on your VM just aren't reachable from the host machine or LAN?)  I'd see at least if you can ping the management IP of your SO VM

**Comment by TotieBash** [comment]
1. Is this new install?
2. Did it ever worked before?
3. Paste the output result of:  `iptables --list -n | grep :443`
4. Paste the output result of:  `cat /opt/so/saltstack/local/pillar/global/soc_global.sls | grep url`
5. What is the IP address of your machine?

**Comment by reyesj2** [comment]
What is the IP you're using for Security Onion and what is the IP of the machine you're wanting to connect to the SOC with? 
Can you share the virtual network info also?

**Comment by ncampbell89** [comment]
I'm just installing it all over again.
